
function practiceSolution(){

}

try {
  module.exports = practiceSolution;
} catch (error) {
  console.log('we caught an error');
}
